cql-parser/src/main/scala/troy/cql/parser/CqlParser.scala

```diff
@@ -53,7 +55,13 @@ object CqlParser extends JavaTokenParsers
    */
   def constant: Parser[Constant] = {
     import Constants._
-    (string | uuid | number | boolean) ^^ Constant // | hex // TODO
+    def str = (string | uuid) ^^ StringConstant
+    def int = integer ^^ { s => new IntegerConstant(s.toInt) }
+    def floatNum = float ^^ { s => new FloatConstant(s.toFloat) }
+    //def uuid = uuid ^^ { s => new UuidConstant(new UUID (UUID.fromString(s))) }
```

Comment:
The input should be String but uuid is an Expression 
"[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}".r